DOCS/man/options.rst

```diff
@@ -3135,6 +3135,11 @@ Window
     Play video with window border and decorations. Since this is on by
     default, use ``--no-border`` to disable the standard window decorations.
 
+``--title-bar``, ``--no-title-bar``
+    Play video with window title bar. Since this is on by default,
+    use ``--no-title-bar`` to hide title bar. ``--no-border`` option takes
+    precedence. (Windows only)
```

Comment:
```suggestion
``--title-bar``, ``--no-title-bar``
    (Windows only)
    Play video with window title bar. Since this is on by default,
    use ``--no-title-bar`` to hide title bar. ``--no-border`` option takes
    precedence.
```
Seems to be the convention (well mostly).

Reply:
In this section of manual it seems to be at the first line. I would change it in separate commit. Look at those `(macOS only)` and `(Windows only)`

Reply:
What do you mean? Since you're adding this, I was just suggesting that the `(Windows only)` should be on the first line like the rest.

Reply:
ah, I'm tired. I read it other way around.

Reply:
fixed.